src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form action="#" method="get">
+        <div class="input-wrapper__first">
+          <img class="input-img" src="images/Search.svg"
+          alt="Search-image">
+          <input
+            data-qa="big keypress"
```

Comment:
```suggestion
            data-qa="keypress"
```
data-qa should be `keypress`. it will solve problems with `element not found` in tests
![Screenshot from 2024-02-04 15-09-48](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/6ee3feed-938d-43b9-9a86-451079f6aee7)